apis/beacon/pool/voluntary_exists.yaml

```diff
@@ -0,0 +1,42 @@
+get:
+  operationId: getPoolVoluntaryExits
+  summary: Get SignedVoluntaryExit from operations pool
+  description: Retrieves voluntary exits known by the node but not necessarily incorporated into any block
+  tags:
+    - Beacon
+  responses:
+    "200":
+      description: Successful response
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: '../../../beacon-node-oapi.yaml#/components/schemas/SignedVoluntaryExit'
+    "500":
+      $ref: '../../../beacon-node-oapi.yaml#/components/responses/InternalError'
+
+post:
+  operationId: submitPoolVoluntaryExit
+  summary: Submit SignedVoluntaryExit object to node's pool
+  description: Submits SignedVoluntaryExit object to node's pool, if all requirements are fulfilled node MUST broadcast it to network.
+  tags:
+    - Beacon
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '../../../beacon-node-oapi.yaml#/components/schemas/SignedVoluntaryExit'
+  responses:
+    "200":
+      description: Voluntary exit is stored in node and broadcasted to network
+    "204":
+      description: Voluntary exit is stored in node and it will be broadcast once all requirements are fulfilled. [See voluntary_exit topic validation](https://github.com/ethereum/eth2.0-specs/blob/v0.11.3/specs/phase0/p2p-interface.md#global-topics)
```

Comment:
So this 204 puts a new requirement on nodes -- to store messages locally and broadcast them on certainly conditions. I'm pretty sure nodes don't hvae this functionality today and I'm not certain it's worth putting it in there.

Reply:
Yeah,makes sense. It's easy to validate epoch constraint against node

Reply:
nit: 'and broadcast to network'